pipeline/restart_node.sh

```diff
@@ -441,6 +441,16 @@ wait_for_harmony_process_to_exit() {
 	done
 }
 
+function cleanup_exp_db
+{
+	# clean up explorer db if exist
+	exp_db="/home/ec2-user/explorer_storage_*"
+	if [ -e "$exp_db" ] ;then
```

Comment:
this may not work, as you have _* at the end.  you need to test it.

Reply:
I tested it out on an exp node, it seemed work fine.